src/parser/priest/shared/modules/shadowlands/covenants/UnholyNova.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import ItemDamageDone from 'interface/ItemDamageDone';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import COVENANTS from 'game/shadowlands/COVENANTS';
+import Events, { ApplyDebuffEvent, CastEvent, DamageEvent, HealEvent } from 'parser/core/Events';
+import ItemHealingDone from 'interface/ItemHealingDone';
+
+const APPLICATION_THRESHOLD = 5000;
+
+// Holy: https://www.warcraftlogs.com/reports/MtraPqxwdB4hRG7j#fight=2
+// Shadow: https://www.warcraftlogs.com/reports/KVDfG2wnb8pABJhj#fight=45
+// Disc: https://www.warcraftlogs.com/reports/GWPC9kQ41yg6z8Xx#fight=47
+class UnholyNova extends Analyzer {
+  totalDamage = 0;
+  totalHealing = 0;
+  totalOverHealing = 0;
+
+  castEvents: CastEvent[] = [];
+  applicationEvents: ApplyDebuffEvent[] = [];
+
+  get totalCasts(): number {
+    return this.castEvents.length;
+  }
+
+  get totalApplications(): number {
+    return this.applicationEvents.length;
+  }
+
+  get totalMisses(): number {
+    let totalMisses = 0;
+    for(const castEvent of this.castEvents) {
+      let didHit = false;
+      for (const applicationEvent of this.applicationEvents){
+        if (Math.abs(applicationEvent.timestamp - castEvent.timestamp) < APPLICATION_THRESHOLD){
+          didHit = true;
+          break;
+        }
+      }
+      if (!didHit) {
+        totalMisses += 1;
+      }
+    }
+    return totalMisses;
+  }
```

Comment:
A suggestion to better place/time their casts would be good to have if they are missing their hits.

Reply:
<img width="1290" alt="Screen+Shot+2020-12-28+at+12 06 52+PM" src="https://user-images.githubusercontent.com/716498/103234452-5387b980-4905-11eb-8226-a9abc8fc1386.png">

Suggestion added.